sonar/modules/documents/api.py

```diff
@@ -359,6 +359,26 @@ def is_open_access(self):
 
         return True
 
+    def is_masked(self):
```

Comment:
it could be a property to avoid using `()` when call it

Reply:
Done